Makefile

```diff
@@ -85,12 +85,8 @@ lint-dockerfile:
 	@hack/lint-dockerfile
 .PHONY: lint-dockerfile
 
-.INTERMEDIATE: Dockerfile.dev
-Dockerfile.dev: Dockerfile Dockerfile.centos.patch
-	patch -o Dockerfile.dev Dockerfile Dockerfile.centos.patch
-
 image: .output/image
-.output/image: Dockerfile.dev $(GO_FILES) $(MANIFEST_FILES) $(OTHER_FILES)
+.output/image: $(GO_FILES) $(MANIFEST_FILES) $(OTHER_FILES)
```

Comment:
can you add the dockerfile generation as a requirement for this?

Reply:
updated